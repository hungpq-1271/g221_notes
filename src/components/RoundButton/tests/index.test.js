import React from 'react'
import { render } from '@testing-library/react'
import 'jest-styled-components'

import RoundButton from '../index'

describe('<RoundButton />', () => {
  it('should render and match the snapshot', () => {
    const { container } = render(<RoundButton>.</RoundButton>)
    expect(container.firstChild).toMatchSnapshot()
  })
})
