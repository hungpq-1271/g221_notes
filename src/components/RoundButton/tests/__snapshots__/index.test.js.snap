// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<RoundButton /> should render and match the snapshot 1`] = `
.c0 {
  background: linear-gradient(45deg,#fe6b8b 30%,#ff8e53 90%);
  border-radius: 100%;
  border: 0;
  color: white;
  height: 48px;
  width: 48px;
  box-shadow: 0 3px 5px 2px rgba(255,105,135,0.3);
}

<button
  class="MuiButtonBase-root MuiButton-root c0 MuiButton-text"
  tabindex="0"
  type="button"
>
  <span
    class="MuiButton-label"
  >
    .
  </span>
  <span
    class="MuiTouchRipple-root"
  />
</button>
`;
